src/core/worker.js

```diff
@@ -458,6 +363,7 @@ var WorkerMessageHandler = {
     // This context is actually holds references on pdfManager and handler,
     // until the latter is destroyed.
     var pdfManager;
+    var fullRequest;
```

Comment:
save only onReaderDocProgress, if needed, but I prefer to not do a round trip from main thread, to just send the same data back.